boards/arm/nrf5340dk_nrf5340/Kconfig

```diff
@@ -43,9 +43,9 @@ config BOARD_ENABLE_CPUNET
 	  may already have started the Network MCU.
 	default y if (BT || NET_L2_IEEE802154)
 
-endif # BOARD_NRF5340PDK_NRF5340_CPUAPP || BOARD_NRF5340PDK_NRF5340_CPUAPPNS
+endif # BOARD_NRF5340PDK_NRF5340_CPUAPP || BOARD_NRF5340PDK_NRF5340_CPUAPPNS || BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPPNS
 
-if BOARD_NRF5340PDK_NRF5340_CPUNET
+if BOARD_NRF5340PDK_NRF5340_CPUNET || BOARD_NRF5340DK_NRF5340_CPUNET
```

Comment:
Same goes for those, can we make `BOARD_NRF5340XDK_NRF5340_CPUNET` and  `BOARD_NRF5340XDK_NRF5340_CPUAPP`?

Reply:
Right, this is an option, I just want us to evaluate, if this is really worth the effort, since we'll eventually clean up the PDK support.

Reply:
Yeah, I think we'll end up removing the support for the PDK as you say, I am convinced this is not worth it right now. Let's get this in as it is and if we want to clean it up later we can always do that.